src/appleseed/foundation/math/beziercurve.h

```diff
@@ -72,19 +72,35 @@ class BezierCurveBase
     typedef Vector<T, 3> VectorType;
     typedef AABB<T, 3> AABBType;
     typedef Matrix<T, 4, 4> MatrixType;
+    typedef Color<T, 3> ColorType;
```

Comment:
You need to include `foundation/image/color.h`.

Reply:
fixed

Reply:
Fixed